### PR TITLE
changing anon-stats from panopticlick to coveryourtracks

### PR DESCRIPTION
**Pull request by lschatzkin**

### CODE REVIEW

File: templates/layout.html [hunk 1]
```diff
@@ -89,11 +89,21 @@ <h2 class="tagline">
...
-      <div style="height: 0; width: 0; position: absolute" id="anon-stats"></div>
+      <!-- Matomo -->
       <script type="text/javascript">
-        document.getElementById('anon-stats').innerHTML = '<img src="https://{{ config.get('MATOMO_URL') }}/js/?idsite={{ config.get('MATOMO_SITE_ID') }}&amp;rec=1&amp;urlref=' + encodeURIComponent(document.referrer) + '&amp;action_name=' + encodeURIComponent(document.title) + '" style="border:0" height="0" width="0" alt="" />';
+        var _paq = window._paq = window._paq || [];
+        /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+        _paq.push(['trackPageView']);
+        _paq.push(['enableLinkTracking']);
+        (function() {
+          var u="https://anon-stats.eff.org/";
```
Comment (Hainish) [inline templates/layout.html:99]: This needs to be generalized to `config.get('MATOMO_URL')`

File: templates/layout.html [hunk 2]
```diff
@@ -89,11 +89,21 @@ <h2 class="tagline">
...
+        _paq.push(['setTrackerUrl', u+'piwik.php']);
+        _paq.push(['setSiteId', '40']);
```
Comment (Hainish) [inline templates/layout.html:101]: This needs to be generalized to `config.get('MATOMO_SITE_ID')`

**Review by Hainish — CHANGES_REQUESTED** [review]
See comments - thanks!

File: templates/layout.html (hunk 2 above)
Comment (lschatzkin) [reply templates/layout.html:101]: Okay, I usually just use the code provided by anon-stats, but it seems I should leave the code that was already there.  I'm not understanding where the MATOMO_SITE_ID AND MATAMO_URL are being set though. I see it referenced in main.py and env_config.py.

Comment (Hainish) [reply templates/layout.html:101]: It should be set in the deployment repo environmental variables, I'll message you with the details.

**Pull request closed without merging**